src/Microsoft.AspNet.Mvc.ViewFeatures/Internal/MvcViewOptionsSetup.cs

```diff
@@ -14,16 +17,24 @@ public class MvcViewOptionsSetup : ConfigureOptions<MvcViewOptions>
         /// <summary>
         /// Initializes a new instance of <see cref="MvcViewOptionsSetup"/>.
         /// </summary>
-        public MvcViewOptionsSetup()
-            : base(ConfigureMvc)
+        public MvcViewOptionsSetup(IServiceProvider serviceProvider)
```

Comment:
[Question](https://github.com/aspnet/Mvc/pull/3200/files#r40466825) also applies here since it's pas through.